test/integration_test/migration_test.go

```diff
@@ -2027,6 +2032,288 @@ func transformResourceTest(t *testing.T) {
+	namespace := fmt.Sprintf("%s-%s", appKey, instanceID)
+
+	var err error
+	defer func() {
+		err = setSourceKubeConfig()
+		require.NoError(t, err, "Error resetting source config")
+	}()
+
+	ctxs, preMigrationCtx := triggerMigration(
+		t,
+		instanceID,
+		appKey,
+		[]string{},
+		[]string{instanceID},
+		false,
+		false,
+		true,
+		false,
+		"",
+		nil)
+
+	err = schedulerDriver.WaitForRunning(ctxs[0], defaultWaitTimeout/2, defaultWaitInterval)
+	require.NoError(t, err, "Migration could not be completed")
+
+	// Change kubeconfig to destination
+	err = setDestinationKubeConfig()
+	require.NoError(t, err, "failed to set kubeconfig to source cluster: %v", err)
+
+	destDeployments, err := apps.Instance().ListDeployments(namespace, meta_v1.ListOptions{})
+	require.NoError(t, err, "error retriving deployments list from %s namespace", namespace)
+	require.Equal(t, 0, len(destDeployments.Items), fmt.Sprintf("Expected no deployments in destination in %s namespace", namespace))
+	destServices, err := core.Instance().ListServices(namespace, meta_v1.ListOptions{})
+	require.NoError(t, err, "error retriving services list from %s namespace", namespace)
+	require.Equal(t, 0, len(destServices.Items), fmt.Sprintf("Expected no service in destination in %s namespace", namespace))
+
+	destroyAndWait(t, []*scheduler.Context{preMigrationCtx})
+
+	err = setSourceKubeConfig()
+	require.NoError(t, err, "failed to set kubeconfig to source cluster: %v", err)
+
+	destroyAndWait(t, ctxs)
+
+	blowNamespacesForTest(t, instanceID, appKey, false)
+	// If we are here then the test has passed
+	testResult = testResultPass
+	logrus.Infof("Test status at end of %s test: %s", t.Name(), testResult)
+}
+
+func excludeResourceTypesWithSelectorsTest(t *testing.T) {
+	var testrailID, testResult = 93466, testResultFail
+	runID := testrailSetupForTest(testrailID, &testResult)
+	defer updateTestRail(&testResult, testrailID, runID)
+	instanceID := "exclude-resourcetype-selector"
+	appKey := "mysql-1-pvc"
+	namespace := fmt.Sprintf("%s-%s", appKey, instanceID)
+
+	var err error
+	defer func() {
+		err = setSourceKubeConfig()
+		require.NoError(t, err, "Error resetting source config")
+	}()
+
+	ctxs, preMigrationCtx := triggerMigration(
+		t,
+		instanceID,
+		appKey,
+		[]string{},
+		[]string{instanceID},
+		false,
+		false,
+		true,
+		false,
+		"",
+		nil)
+
+	err = schedulerDriver.WaitForRunning(ctxs[0], defaultWaitTimeout/2, defaultWaitInterval)
+	require.NoError(t, err, "Migration could not be completed")
+
+	// Change kubeconfig to destination
+	err = setDestinationKubeConfig()
+	require.NoError(t, err, "failed to set kubeconfig to source cluster: %v", err)
+
+	destDeployments, err := apps.Instance().ListDeployments(namespace, meta_v1.ListOptions{})
+	require.NoError(t, err, "error retriving deployments list from %s namespace", namespace)
+	require.Equal(t, 1, len(destDeployments.Items), fmt.Sprintf("Expected 1 deployment in destination in %s namespace", namespace))
```

Comment:
Shouldn't we have PVC in this list too?

Reply:
+1

Reply:
yes, added the check for other resources.